tests/cli/add/conftest.py

```diff
@@ -6,13 +6,13 @@
 import pytest
 
 
-@pytest.fixture(name="file_data")
-def fixture_file_data(second_sample_vcf: Path, case_id: str, sample_tag_names: List[str]) -> dict:
+@pytest.fixture()
```

Comment:
I think `@pytest.fixture` is enough, the parentheses are not required when no arguments are given

Reply:
Good suggestion - fixed